### ISSUE TITLE
Not able to run the repo

### ISSUE THREAD

**Issue by amarnathk1547** [christian-bromann/webdriverio-performance-testing#1]
Hi @christian-bromann,

It's a wonderful idea to communicate with dev-tools and analyze the performance of the web applications using v5. I am really excited to see how this works. But the sad part is that I am not able to run any of the spec files.

Checked selenium-standalone module as well, but concatenation of the url is not broken at that point. Not sure from where exactly wdio is trying to phrase the url and hit. Please advise.

Attaching the log files here. The only issue I could see as of now is sending the POST request to the invalid url. I mean http://127.0.0.1/:4444E/wd/hub/session

[wdio-0-0.log](https://github.com/christian-bromann/webdriverio-performance-testing/files/2364006/wdio-0-0.log)
[wdio-0-1.log](https://github.com/christian-bromann/webdriverio-performance-testing/files/2364007/wdio-0-1.log)
[wdio.log](https://github.com/christian-bromann/webdriverio-performance-testing/files/2364008/wdio.log)

**Comment by christian-bromann** [comment]
It looks like chromedriver is not started where the request are send to. Please open a bug ticket in the [selenium-standalone](https://www.npmjs.com/package/selenium-standalone) package as this seems to be clearly a mis-function when running stuff in Windows.

For the time being you can run Chromedriver manually and set host and port in your wdio.conf.js.

**Comment by amarnathk1547** [comment]
I have started the chromedriver like this..

`java -Dwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone-3.14.0.jar`

The above started by default on 4444 port. The changes I have made to config part is 
```
//runner: 'local',
    host: '127.0.0.1',
    port: 4444,
```
Then selenium-standalone logs like this

> 16:28:09.351 ERROR [SeleniumServer.boot] - Port 4444 is busy, please choose a free port and specify it using -port option

Now I removed selenium-standalone from services and manually started it on 4444 port with chromedriver. Still no luck. The error message remains the same and the test fails. Any comments?
